extensions/gitlab/src/components/issues.tsx

```diff
@@ -280,7 +281,24 @@ export function IssueList({
       onSearchTextChange={setSearchText}
       isLoading={isLoading}
       throttle={true}
-      searchBarAccessory={searchBarAccessory}
+      searchBarAccessory={
+        <List.Dropdown
+          tooltip="State"
+          onChange={(newValue) => {
+            for (const value of Object.values(IssueState)) {
+              if (value === newValue) {
+                setSearchState(IssueState[newValue]);
```

Comment:
Please set the searchText which already support this kind of filter. That way it is still combineable as in the GitHub extension

Reply:
[Line 281](https://github.com/raycast/extensions/pull/10513/files#diff-d21704d6f1468a42f3d39ed8bf40f4eba5df3dd84980e42620388a9fe4251008R281) is already doing that, isn’t it?

Reply:
No. It just change the state. I see that I don't have the tokenizer in this component, so it would be harder for you.
I will add that at a later point.
The idea to use the dropbox to set the text in the searchbar automatically like it is done in the GitLab search. That way it is still combine-able.